app/src/common/shared/com/igalia/wolvic/utils/UrlUtils.java

```diff
@@ -208,6 +208,26 @@ public static boolean isAddonsUrl(@Nullable String url) {
         return url.equalsIgnoreCase(ABOUT_ADDONS);
     }
 
+    public static final String ABOUT_WEBAPPS = "about://webapps";
```

Comment:
Are those special URIs valid for Chromium backend? I guess they are not. We should be careful not to include that in the `common/shared` dir.

Reply:
No, they are not valid for Chromium. I assumed that the code would intercept these URL and show the appropriate library panel, just like it happens with Gecko.

By the way, I think that our Gecko URLs are not valid either; for example, Firefox uses `about: history` and not `about://history`